lib/nimble_template/helpers/generator.ex

```diff
@@ -24,7 +24,12 @@ defmodule NimbleTemplate.Generator do
 
   def rename_file(old_path, new_path), do: File.rename(old_path, new_path)
 
-  def replace_content(file_path, anchor, content, raise_exception \\ true) do
+  def replace_content(file_path, anchor, content, raise_exception \\ true)
+
+  def replace_content(_file_path, anchor, content, _raise_exception) when anchor == content,
```

Comment:
```suggestion
  def replace_content(_file_path, anchor, anchor, _raise_exception),
```
I think this also works but yours I think is more clear what it's doing? 👁️

Reply:
that isn't clear enough I think @liamstevens111 and I'm not sure it work as the same 😂 so let go with `when anchor == content`

Reply:
Yeah should work the same but I agree, probably better to be explicit here @andyduong1920